res/layout/conversation_item_received.xml

```diff
@@ -127,6 +127,22 @@
                     android:contentDescription="@string/conversation_item_sent__message_pinned_description"
                     tools:visibility="visible"/>
 
+                <ImageView
+                    android:id="@+id/read_reminder"
+                    android:layout_width="12dp"
+                    android:layout_height="11dp"
+                    android:src="@drawable/icon_notification"
```

Comment:
does this mean the icon will be blue next to a bunch of grey icons?

Reply:
It's actually a completely different Icon. I will remove the exclamation point image for this pr, but the image is now the Signal message bubble, which gets tinted the same grey as the rest of the icons